packages/components/src/components/toast/toast-item/toast-item-styles.ts

```diff
@@ -7,15 +7,30 @@ import {
   getNotificationRootJssStyle,
 } from '../../inline-notification/inline-notification-styles-shared';
 import { dropShadowHighStyle, textSmallStyle } from '@porsche-design-system/utilities-v2';
-import { getThemedColors, prefersColorSchemeDarkMediaQuery } from '../../../styles';
+import { addImportantToEachRule, getThemedColors, prefersColorSchemeDarkMediaQuery } from '../../../styles';
 
 export const getComponentCss = (state: ToastState, theme: Theme): string => {
   return getCss({
     '@global': {
       ':host': {
         display: 'grid',
+        opacity: 0, // needed to prevent flickering on initial render
+        maxWidth: 'inherit',
+        boxSizing: 'border-box',
```

Comment:
why are some styles not wrapped into important helper?

Reply:
Adapted. But `opacity` needs to be overwritable due to the animation of the parent Toast element.